mpd/client.py

```diff
@@ -217,168 +217,93 @@ def _init():
 
     return client, conn, cur, current_song_coords
 
-
-def main_album(queue_length):
+def main_album(queue_length, option_best):
```

Comment:
Maybe we can use `option_best=True` to keep the previous behavior as default for this function?